spec/unit/webrtc/groupCall.spec.ts

```diff
@@ -545,5 +568,87 @@ describe('Group Call', function() {
                 groupCall.terminate();
             });
         });
+
+        describe("Incoming Calls", function() {
+            it("ignores incoming calls for other rooms", async () => {
+                const groupCall = await createAndEnterGroupCall(mockClient, room);
```

Comment:
Extract this into a `beforeEach` and `groupCall.leave()` into an `afterEach` since they're the same for every test in the block?